### PR TITLE
Manish/instrumentingfor logs

### PR DESCRIPTION
**Pull request by ranjanmanish**

### CODE REVIEW

File: src/components/trafficFlow.jsx [hunk 1]
```diff
@@ -122,11 +122,26 @@ class TrafficFlow extends React.Component {
     this.setState({ currentView: currentView });
   }
 
+  fetchData () {
+    request.get('http://192.168.33.10:7880/vizc')
```
Comment (ranjanmanish) [inline src/components/trafficFlow.jsx]: This will not be hard coded. Will address this as a part of config

**Comment by aaronblohowiak** [comment]
This seems like a work in progress, are you sure you wanted to open a PR already?

**Comment by ranjanmanish** [comment]
Naah, Sorry.  please  reject it. I was trying sth out. ended up creating this rather than pushing to my local.

**Comment by aaronblohowiak** [comment]
Ok. I like where this is going and would love a more pluggable example app. Let me know!

**Pull request closed without merging**